### PR TITLE
Updated volunteer page copy

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes https://github.com/MozillaFoundation/2016.Mozillafestival.org/issues/112

@alanmoo tiny PR for you to review plz 😄

### CODE REVIEW

File: less/pages/volunteer.less [hunk 1]
```diff
@@ -2,7 +2,7 @@
   .hero-unit-container {
     background: linear-gradient( to right, #4A825D, #406882);
   }
-  .thank-you {
```
Comment (mmmavis) [inline less/pages/volunteer.less]: can't find any `thank-you` class element in the codebase. this is probably some old code that we forgot to remove

**Comment by mmmavis** [comment]
@alanmoo yea i wasn't sure if it was our intention

**Comment by SpikeUK** [comment]
Thank you @mmmavis and @alanmoo !

**Comment by mmmavis** [comment]
@SpikeUK 😃

**Pull request merged**